src/main/java/racingcar/Cars.java

```diff
@@ -0,0 +1,68 @@
+package racingcar;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Cars {
+    private static final String CAR_NAME_DELIMITER = ",";
+    public static final int DEFAULT_POSITION = 0;
+
+    private final List<Car> cars;
+
+    public Cars(String s) {
+        this(Arrays.asList(s.split(CAR_NAME_DELIMITER)));
+    }
+
+    public Cars(List<String> carNames) {
+        validateNotDuplicated(carNames);
+        this.cars = carNames.stream().map(Car::new).collect(Collectors.toList());
+    }
+
+    private void validateNotDuplicated(List<String> carNames) {
+        if (carNames.size() != carNames.stream().distinct().count()) {
+            throw new IllegalArgumentException("중복된 자동차 이름이 존재합니다.");
+        }
```

Comment:
적절히 변수로 뽑아주는 것이 조금 더 가독성이 좋을 것 같아요. 
```suggestion
        var distinctCarSizes = carNames.stream()
            .distinct()
            .count();
        if (carNames.size() != distinctCarSizes) {
            throw new IllegalArgumentException("중복된 자동차 이름이 존재합니다.");
        }
```

혹은 아래와 같이 자료구조의 특성을 이용한 방법도 있을 것 같네요. 🙂
```java
        if (carNames.size() != new HashSet<>(carNames).size()) {
            throw new IllegalArgumentException("중복된 자동차 이름이 존재합니다.");
        }
```